src/main/java/tuteez/ui/PersonCard.java

```diff
@@ -99,4 +106,20 @@ private void setTelegramUsernameText(Person person) {
             telegram.setVisible(false);
         }
     }
+
+    /**
+     * Displays the next lesson scheduled for the {@code Person}, based on the current time.
+     * If no lessons are scheduled, hides the lessons flow pane.
+     *
+     * @param person The {@code Person} whose next lesson is to be displayed.
+     */
+    private void setNextLesson(Person person) {
+        Lesson nextLesson = person.nextLessonBasedOnCurrentTime();
+        if (nextLesson != null) {
+            lessons.getChildren().add(new Label(person.nextLessonBasedOnCurrentTime().getDayAndTime()));
```

Comment:
You already have a `nextLesson` variable, is there a reason why you call the `nextLessonBasedOnCurrentTime` again? Lmk if I have missed something :))

Reply:
Also perhaps it is more appropriate to call the flow pane `nextLessonPane` rather than `lessons`, the latter could imply that we intend to display more than one lesson on the left side

Reply:
Good catch! I have edited my code in the latest commit!